js/signin.js

```diff
@@ -0,0 +1,104 @@
+const emailInput = document.querySelector("#email-input");
+const passwordInput = document.querySelector("#password-input");
+const emailErrorMessage = document.querySelector("#email-error-message");
+const passwordErrorMessage = document.querySelector("#password-error-message");
+const loginForm = document.querySelector("#login-form");
+
+const eyeOffButton = document.querySelector(".eye-off");
+const eyeOnButton = document.querySelector(".eye-on");
+
+const EMAIL_REGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/;
+
+// 이메일 입력
+emailInput.addEventListener("focusout", checkEmailInput);
+emailInput.addEventListener("focus", resetInputStyle);
+
+function checkEmailInput() {
+  if (emailInput.value.trim() === "") {
+    showEmailErrorMessage("이메일을 입력해 주세요.");
+  } else if (!validateEmail(emailInput.value.trim())) {
+    showEmailErrorMessage("올바른 이메일 주소가 아닙니다.");
+  } else {
+    hideErrorMessage();
+  }
+}
+
+function validateEmail(email) {
+  return EMAIL_REGEX.test(email);
+}
+
+function showEmailErrorMessage(message) {
+  emailErrorMessage.textContent = message;
+  emailErrorMessage.style.display = "block";
+  emailInput.classList.add("sign-input-error");
+}
+
+function hideErrorMessage() {
+  emailErrorMessage.style.display = "none";
+  emailInput.classList.remove("sign-input-error");
+}
+
+function resetInputStyle() {
+  hideErrorMessage();
+}
```

Comment:
`resetInputStyle`라는 함수는 `hideErrorMessage`를 한번 감싸놓는 역할만 하고 있는데요~ `hideErrorMessage`를 그대로 사용해주는 건 어떨까요?

물론 여기서 `resetInputStyle`은  의미 있게 만들어진 함수라고 생각합니다!

Reply:
아래쪽에 `resetPasswordInputStyle`와 동일하게 형식을 가져가려면 Email을 넣어주는 게 좋을 것 같아요!

```suggestion
function resetEmailInputStyle() {
  hideErrorMessage();
}
```